torch/distributions/transforms.py

```diff
@@ -807,7 +807,7 @@ def log_abs_det_jacobian(self, x, y):
         if isinstance(scale, _Number):
             result = torch.full_like(x, math.log(abs(scale)))
         else:
-            result = scale.abs().log()
+            result = torch.abs(scale).log()
```

Comment:
No longer needed; thanks to #145674 mypy correctly infers Tensor in the else branch.